### ISSUE TITLE
3005 Not found (but user exists.)

### ISSUE THREAD

**Issue by XizzoR** [wazuh/wazuh-kibana-app#1108]
Hello there!

I am using Wazuh v3.7.1 and Kibana 6.5.1.
Every component of Elastic stack is on its own Docker container, also Wazuh have his own container. Every container is in same docker network. Kibana have exposed and published port 5601, when Wazuh have published and exposed ports 1514, 1515, 514 and 55000.

I am using XPack. I made a role for managing all wazuh incides (along with "read" permissions in Kibana), assign it to manually made "wazuh-api" user and use "configure_api.sh" on Wazuh to generate API on 55000 port with same user/pass as "wazuh-api" user.

Whole Elastic stack, Wazuh and Kibana's Wazuh App works without errors (there is no error messages in logs). After filling "API configuration" form with user/pass (of "wazuh-api" user), url as "https://hostip" where "hostip" is IP address of Ubuntu machine hosting all of the Docker containers, and port as 5601 (since port 55000 gives me an error message: "3005 - Wrong port being used to connect to the Wazuh API") but app returns 3005 - Not Found.

kibana.log
```
{"type":"response","@timestamp":"2018-12-23T18:41:59Z","tags":[],"pid":9,"method":"get","statusCode":404,"req":{"url":"/version","method":"get","headers":{"accept":"*/*","user-agent":"Needle/2.2.4 (Node.js v8.11.4; linux x64)","wazuh-app-version":"3.7.1","host":"10.0.13.55:5601","connection":"close"},"remoteAddress":"172.17.0.1","userAgent":"172.17.0.1"},"res":{"statusCode":404,"responseTime":286,"contentLength":9},"message":"GET /version 404 286ms - 9.0B"}
```

wazuhapp.log
```
Dec 23, 2018 7:41:59 PM | ERROR | Not Found
Dec 23, 2018 7:40:59 PM | ERROR | connect ECONNREFUSED 10.0.13.55:55000
```

**Comment by juankaromo** [comment]
Hi @XizzoR, we're going to run some checks.

First of all check that the API is running using this command:

`systemctl status wazuh-api`

You should get a response like this: `Active: active (running)`

On the other hand, if when you configured the api you activated the https to secure it, you must use `https://`, and otherwise `http://`

Let us know the results.

Regards,
Juan Carlos

**Comment by XizzoR** [comment]
I apologise for letting this issue opened for so long, but I solved the problem. Silly me, I left 55000 exposed, but not published (as I initially thought) to the host's 55000 port. Another problem was not using the configure_api.sh properly. 
Off topic, but I saw that Wazuh app is not working if user in kibana.yml is not equal to "elastic". That brings some permission conflicts in my case and it would be pretty cool if we could change that username later (in the app itself or at the "kibana-plugin install" ) to something else.
